### DISCUSSION TITLE
hoe-ziet-voor-jou-een-typische-dag-of-week-eruit

### DISCUSSION THREAD

**Discussion by rutgerkock** [fdnd/docs.fdnd.nl#165] [Reflectie]
Maandag op school
Dinsdag thuis
Woensdag op school
Donderdag thuis
Vrijdag lang op school

**Comment by SamaraFellaDina** [comment]
> Maandag op school Dinsdag thuis Woensdag op school Donderdag thuis Vrijdag lang op school

gespreekt

![](https://media.giphy.com/media/sEqG2g9RGj91PgSRnz/giphy.gif?cid=790b7611ztvggqgj558721c4uftxuwt0f0txwv48qv4e57ry&ep=v1_gifs_trending&rid=giphy.gif&ct=g)

**Comment by Jesse-L3** [comment]
exactly. straight to the point.
